lib/interfaces/common.js

```diff
@@ -2,6 +2,13 @@
 
 var Suite = require('../suite');
 
+function setOnlies (suite) {
+  if ((suite.hasOnly || suite.isOnly) && suite.parent) {
+    suite.parent.hasOnly = true;
+    setOnlies(suite.parent);
+  }
+}
+
```

Comment:
Seeing as this is called with both tests and suites, it is somewhat confusing that the parameter name suggests it is only meant to be called with suites. I also believe it would be preferable to document this function even though it is private.